Apps/Sandcastle/gallery/3D Tiles Point Cloud Styling.html

```diff
@@ -113,6 +113,11 @@
     color : "color() * cos(${temperature} - 0)"
 });
 
+addStyle('Sqrt', {
+    color : "color() * sqrt(${temperature} - 0.5)",
```

Comment:
`temperature` is in the range 0.0 to 1.0 so you should tweak this style to avoid undefined behavior.